static/sass/_pattern_strip.scss

```
@@ -625,7 +625,7 @@
         linear-gradient(251deg, #e95422 0%, #772953 49.5%, #2c001e 100%);
     }
 
-    background-position: top left, top right, center 67%, left top;
+    background-position: top left, top right, center 66.4%, left top;
```

I am seeing the black line now. 66.5 works better for me... test before you accept this.
```suggestion
    background-position: top left, top right, center 66.5%, left top;
```